zaza/openstack/charm_tests/neutron/tests.py

```diff
@@ -118,7 +146,7 @@ def test_900_restart_on_config_change(self):
         """
         # Expected default and alternate values
         current_value = zaza.model.get_application_config(
-            'neutron-gateway')['debug']['value']
+            self._APP_NAME)['debug']['value']
```

Comment:
Where does `_APP_NAME` come from?  I've grepped the code and can't seem to find it.  Is it Juju specific itself?

Reply:
new line 116 of this file :)